app/views/proposals/show.html.haml

```diff
@@ -97,3 +97,10 @@
               %dt Requires Registration:
               %dd
                 = link_to "Yes (#{registered_text(@event)})", new_conference_conference_registration_path(@conference.short_title), class: 'btn btn-xs btn-danger', disabled: !@event.registration_possible?
+          - if concurrent_events(@event).present?
+            .col-md-12#concurrent-events
+              %h4
+                = "Other Events on \n #{@event_schedule.start_time.strftime("%Y %B %e")} at #{@event_schedule.start_time.strftime("%H:%M")}:"
+              - concurrent_events(@event).each do |event|
+                %h4
+                  = link_to event.title, conference_program_proposal_path(@conference.short_title, event.id)
```

Comment:
This might look better if we show it with %li, as some event titles might use more than one line and then it gets confusing